api/src/main/java/org/openmrs/module/idgen/service/IdentifierSourceService.java

```diff
@@ -265,6 +265,14 @@
      */
     @Transactional(readOnly=true)
     IdentifierSource getIdentifierSourceByUuid(String uuid);
+    
+    /**
+     * Queries the database for the identifier source(s) with identifier type: patientIdentifierType 

```

Comment:
The service layer should not mention anything like database because it is the data access layer that knows whether we store in a database, file, network, etc.
Take a look at the javadoc of existing service methods to see how we describe them.

Reply:
fixed